frameworks/hdfs/tools/docker-client/Dockerfile

```diff
@@ -1,37 +1,39 @@
 FROM ubuntu:16.04
 
-RUN apt-get -y update && \
-    apt-get -y upgrade && \
-    apt-get -y install wget software-properties-common
-
-RUN wget https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.9.1-dcos.tar.gz && \
-    tar xf hadoop-2.6.0-cdh5.9.1-dcos.tar.gz && \
-    rm hadoop-2.6.0-cdh5.9.1-dcos.tar.gz
-
-RUN apt-get update && \
-    apt-get install -y default-jre && \
-    apt-get install -y \
-    curl \
-    dnsutils \
-    less \
-    vim
-
 ENV HADOOP_VERSION hadoop-2.6.0-cdh5.9.1
 ENV HDFS_SERVICE_NAME hdfs
 ENV JAVA_HOME /usr/lib/jvm/default-java
 
+RUN apt-get -y update && \
+    apt-get -y upgrade && \
+    apt-get -y install \
+      curl \
+      default-jre \
+      dnsutils \
+      less \
+      software-properties-common \
+      vim \
+      wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN wget https://downloads.mesosphere.com/hdfs/assets/$HADOOP_VERSION-dcos.tar.gz && \
+    tar xf $HADOOP_VERSION-dcos.tar.gz && \
+    rm $HADOOP_VERSION-dcos.tar.gz
+
+
 # the env var is required for a non-interactive install of krb5-user
-RUN export DEBIAN_FRONTEND=noninteractive && apt-get install -y krb5-user
-COPY krb5.conf /etc/krb5.conf
-RUN chown -R nobody /etc
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y krb5-user && \
+    apt-get clean
 
-COPY configure-environment.sh /hadoop-2.6.0-cdh5.9.1/configure-environment.sh
-COPY configure-hdfs.sh /hadoop-2.6.0-cdh5.9.1/configure-hdfs.sh
+COPY krb5.conf /etc/krb5.conf
```

Comment:
Would the default `krb5.conf` that comes with `krb5-user` not be sufficient?

Reply:
It has all the correct fields (and some additional ones that aren't needed since we've simplified our krb5 file to include only fields we need).

The default values of the fields are awkward to correct without writing some wrapper scripts specifically for this manipulation (which is more work than needed).